src/components/UserProfiles/UserProfileBase/index.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+    ArrowLeft as ArrowLeftIcon,
+    Settings as SettingsIcon,
+    ExitToApp as ExitToAppIcon,
+    Edit as EditIcon,
+} from "@material-ui/icons";
+import AvatarBase from "src/components/Avatars/AvatarBase";
+import BoxBase, { BoxBaseProps } from "src/components/Boxs/BoxBase";
+import ButtonBase from "src/components/Buttons/ButtonBase";
+import DividerBase from "src/components/Dividers/DividerBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+
+const replyLogoSource =
+    "https://res.cloudinary.com/dq7l8216n/image/upload/v1621347887/Reply_Logo.png";
+const fcodeAvatarSource =
+    "https://res.cloudinary.com/dq7l8216n/image/upload/v1620235303/FCode-Avatar.png";
+
+export interface UserProfileBaseProps extends BoxBaseProps {
+    variant?: "logged-in" | "logged-out";
+}
+
+const UserProfileBase = (props: UserProfileBaseProps) => {
+    const { variant, ...rest } = props;
+
+    let userProfileLoggedIn: JSX.Element = (
+        <BoxBase {...rest}>

```

Comment:
I don't think we should give the user this much control using ...rest. w={1} is good enough